### DISCUSSION TITLE
Add option to opt in/out of log file creation

### DISCUSSION THREAD

**Discussion by ANN-4-BE-11-A** [ppy/osu#22320] [Ideas] [answered]
In Ubuntu, that'd be the files from `/home/.local/share/osu/logs/`. I suspect said files or their contents are stored elsewhere as well. Cuz the files come back as if nothing happened after opening and closing Osu!. So if that's indeed the case, then opting out should disable this other location too.

I request this feature cuz of privacy and space storage concerns.

**Comment by bdach** [comment, marked as answer]
>I suspect said files or their contents are stored elsewhere as well. Cuz the files come back as if nothing happened after opening and closing Osu!.

They come back because each time you run the game, the log is truncated and written to again. The *game* is logging things. So that when somebody gets a crash, they can attach the logs on github and we can diagnose the issue.

> I request this feature cuz of privacy and space storage concerns.

Respectfully, these arguments make no sense. The files should be kilobytes in size at most, and are not sent elsewhere. The game _does_ phone home to a sentry instance for bug reporting as well, but that is a separate matter entirely.

**Reply by ANN-4-BE-11-A** [reply]
If you say so...

**Reply by bdach** [reply]
You don't have to trust me on this. All of the source is available to read at your discretion.
